src/server/scripts/Commands/cs_account.cpp

```diff
@@ -113,6 +113,10 @@ class account_commandscript : public CommandScript
             handler->SendSysMessage(LANG_ACCOUNT_TOO_LONG);
             handler->SetSentErrorMessage(true);
             return false;
+        case AOR_PASS_TOO_LONG:
+            handler->SendSysMessage(LANG_ACCOUNT_PASS_TOO_LONG);
+            handler->SetSentErrorMessage(true);
+            return false;
```

Comment:
How its related to the PR?

Reply:
No it was my bad, i've created the PR with my fork on master branch, now this PR is merged than this snippet will not be considered.